include/robot_state_publisher/robot_state_publisher.h

```diff
@@ -37,9 +37,10 @@
 #ifndef ROBOT_STATE_PUBLISHER_H
 #define ROBOT_STATE_PUBLISHER_H
 
+#include <map>
+#include <string>
+
 #include <ros/ros.h>
-#include <boost/scoped_ptr.hpp>
-#include <tf/tf.h>
```

Comment:
Removing `<tf/tf.h>` in #82 on `melodic-devel` required ros-naoqi/naoqi_driver#103 . Maybe it should be left in place to avoid breaking source builds of other kinetic/lunar packages?

Reply:
Targeting noetic